prog/weaver/http.go

```diff
@@ -114,6 +116,21 @@ var rootTemplate = template.New("root").Funcs(map[string]interface{}{
 		return "disabled"
 	},
 	"trimSuffix": strings.TrimSuffix,
+	"versionMeta": func(meta *VersionMeta) string {
+		if meta == nil {
+			return ""
+		}
```

Comment:
Theoretically, it can. E.g., when that goroutine https://github.com/weaveworks/weave/blob/issues/2089-vsn-check/prog/weaver/main.go#L194 gets scheduled after the first `weave status|report` request.